_inc/client/rest-api/index.js

```diff
@@ -225,6 +225,11 @@ function JetpackRestApiClient( root, nonce ) {
 				.then( parseJsonResponse )
 				.then( body => JSON.parse( body.data ) ),
 
+		fetchProducts: () =>
+			getRequest( `${ apiRoot }jetpack/v4/products`, getParams )
+				.then( checkStatus )
+				.then( parseJsonResponse ),
+
```

Comment:
Are we missing converting the json data into an json object? with `.then( body => JSON.parse( body.data ) ),` 
here?

Reply:
In this case no, because `body` itself is the JSON object we want and not `body.data`